jbpm-test-coverage/src/test/resources/org/jbpm/test/regression/ISOTimerTest.bpmn2

```diff
@@ -1,150 +1,150 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Just one thing I noticed is that for some reason (probably due to a whitespace) this file was included as well. But it's up to you if you want to revert it :)

Reply:
No... actually it is due to the newline character/line separator, apparently that file was uploaded using CRTL windows style and when I saw it got changed in my laptop to LF linux style - think due to KIE coding standards I have in my Intellij.

So.. I guess it should be using LF instead of CRTL? that's why it is changed, nothing else.

Reply:
Ah, sure, no problem.